cmd/duffle/relocate_test.go

```diff
@@ -2,16 +2,19 @@ package main
 
 import (
 	"encoding/json"
-	"fmt"
+	"errors"
 	"io/ioutil"
 	"os"
 	"path"
 	"path/filepath"
 	"reflect"
+	"strings"
 	"testing"
 
+	"github.com/deislabs/duffle/pkg/imagestore"
+	"github.com/deislabs/duffle/pkg/imagestore/imagestoremocks"
```

Comment:
nit: the go import grouping is weird, there's duffle/home package alone line 19

Reply:
Fixed.